src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackServiceTest.java

```diff
@@ -0,0 +1,189 @@
+import uk.gov.hmcts.reform.bulkscan.orchestrator.config.ServiceConfigItem;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.callback.ExceptionRecordValidator;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.callback.ProcessResult;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.config.ServiceConfigProvider;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.util.Map;
+
+import static java.time.LocalDateTime.now;
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.emptyMap;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.CallbackValidationsTest.JOURNEY_CLASSIFICATION;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.EventIdValidator.EVENT_ID_ATTACH_TO_CASE;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.ExceptionRecordFields.ATTACH_TO_CASE_REFERENCE;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.ExceptionRecordFields.OCR_DATA;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.ExceptionRecordFields.SCANNED_DOCUMENTS;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.domains.envelopes.model.Classification.SUPPLEMENTARY_EVIDENCE_WITH_OCR;
+
+@ExtendWith(MockitoExtension.class)
+class AttachCaseCallbackServiceTest {
+
+    private AttachCaseCallbackService attachCaseCallbackService;
+
+    @Mock
+    private ServiceConfigProvider serviceConfigProvider;
+
+    @Mock
+    private CcdApi ccdApi;
+
+    @Mock
+    private ExceptionRecordValidator exceptionRecordValidator;
+
+    @Mock
+    private CcdCaseUpdater ccdCaseUpdater;
+
+    @Mock
+    private PaymentsProcessor paymentsProcessor;
+
+    private static final String JURISDICTION = "BULKSCAN";
+    private static final String CASE_TYPE_EXCEPTION_RECORD = "BULKSCAN_ExceptionRecord";
+    private static final String CASE_REF = "1539007368674134";
+    private static final String DOCUMENT_FILENAME = "document.pdf";
+    private static final String DOCUMENT_NUMBER = "123456";
+    private static final Map<String, Object> EXISTING_DOC = document(DOCUMENT_FILENAME, DOCUMENT_NUMBER);
+
+    private static final String IDAM_TOKEN = "IDAM_TOKEN";
+    private static final String USER_ID = "USER_ID";
+    private static final String EXISTING_CASE_ID = "12345";
+
+    private ExceptionRecord exceptionRecord;
+    private ServiceConfigItem configItem;
+
+    private static final Map<String, Object> CASE_DATA = ImmutableMap.of(
+        "case_type_id", CASE_TYPE_EXCEPTION_RECORD,
```

Comment:
`"case_type_id"` is a property of `CASE_DETAILS` (below). no need to duplicate this